packages/cxl-ui/scss/cxl-card.scss

```diff
@@ -1,6 +1,6 @@
 :host {
   display: block;
-  padding: var(--lumo-space-m);
+  padding: var(--lumo-space-m) var(--lumo-space-l);
```

Comment:
Good point. To prevent that, the new styles for the dashboard are scoped under `theme="cxl-course"`, so it doesn't affect the current design instances at all. 

But there's a similar issue in the other direction, meaning the new design is not 100% correct, as some tokens would have to be updated globally, like border-radius. I figured overriding it temporarily is unnececessary.

Reply:
While it more be more expedient to introduce it in a feature PR, it affects the visual output of other PR. This change is fairly benign, though in the future we should consider isolating changes.